main_routers/characters_router.py

```
@@ -330,33 +359,142 @@ async def update_catgirl_l2d(name: str, request: Request):
+        # 加载当前角色配置
+        _config_manager = get_config_manager()
+        characters = _config_manager.load_characters()
+
+        # 确保猫娘配置存在
+        if '猫娘' not in characters or name not in characters['猫娘']:
+            return JSONResponse(content={
+                'success': False,
+                'error': '角色不存在'
+            }, status_code=404)
+
+        # 检查是否为VRM模型（可选验证）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)